ndmg/graph/graph.py

```diff
@@ -94,15 +97,14 @@ def make_graph(self, streamlines, attr=None):
                 else:
                     pass
 
-                if loc:
-                    p.add(loc)
-
-            edges = combinations(p, 2)
+            if loc:
+                p.add(loc)
+            edges = set([tuple(sorted(x)) for x in product(p, p)])
             for edge in edges:
-                lst = tuple([int(node) for node in edge])
-                self.edge_dict[tuple(sorted(lst))] += 1
+                lst = tuple(sorted([str(node) for node in edge]))
```

Comment:
Why are you making the nodes strings, and sorting them as strings rather than as numbers? Alphabetic sorting != numeric sorting

Reply:
@ebridge2  does you code work for both alpha and numeric vertex IDs? in general, i am against engineering for people doing dumb stuff.  i'd strongly prefer that we require atlases in a provided and reasonable spec, and provide an error if it is not in that spec.  lest things could have weird behaviors that we don't understand and are not testing.

Reply:
I agree with @jovo here - I think parcellations should be supplied in integer-labelled niftis, which is the standard afaict. If the atlas isn't in this format, it would make more sense to exit with an error.